### ISSUE TITLE
Create new general 'help' entry in Help menu / add a new 'manual' one proper to Pd's manual

### ISSUE THREAD

**Issue by porres** [pure-data/pddp#15]
This can be a quick overview of 'Pd Help'/Documentation, a 'Welcome to Pd' section of some sort. We can write a prototype here: https://nc.nubegris.com.ar/index.php/s/5z4GMqa5HKdj3Lm

prior discussion in https://github.com/pure-data/pddp/issues/13

**Comment by porres** [comment]
I already wrote a preliminary prototype in that link. I'm basically referring to and explaining all options in the Help Menu, plus telling people to join the mailing list for help as well.

Now, currently, besides the mailing list, there's also a facebook group, a telegram group, a discord channel and the old Pd forum. I guess we can refer to all of these options as well here and maybe somewhere else (in Pd's manual in puredata.info)

**Comment by Lucarda** [comment]
>  besides the mailing list, there's also a facebook group, a telegram group, a discord channel and the old Pd forum. I guess we can refer to all of these options as well

Yes I think this is where they should go. also https://forum.pdpatchrepo.info/

**Comment by umlaeute** [comment]
wouldn't it be easier to just link to https://puredata.info/community : this way we can update the list without needing to change the Pd sources.

the page itself doesn't look super modern, but we/I could provide a more generic link that redirects to whatever page we want)

**Comment by Lucarda** [comment]
> wouldn't it be easier to just link to https://puredata.info/community : this way we can update the list without needing to change the Pd sources.

agree.

> the page itself doesn't look super modern, but we/I could provide a more generic link that redirects to whatever page we want)

we should consider creating a https://pages.github.com/ for `pure-data/pure-data` if we need a better looking (just like the manual?) to start including latest updates (of whatever release) not touching the source code (wich i think is nicer). This could be a nice place for things we plan to add on https://github.com/pure-data/pddp/issues/14.

**Comment by Lucarda** [comment]
May be if we have the "github pages" it makes more sence to have the ability to edit files in `.md` and compile them to `.html` like in https://github.com/pure-data/pddp/issues/10 with `pandoc`

**Comment by umlaeute** [comment]
> we should consider creating a https://pages.github.com/ 

i'm not super-excited about tying more of Pd's services to github (i understand that https://puredata.info is not exactly fast to adopt and a beast to overhaul, but i'm a hard believer in self-hosted internet)

> EDIT: Having the .css file in a fixed URL helps a lot.

i don't know what that means in this context.

**Comment by Lucarda** [comment]
> but i'm a hard believer in self-hosted internet

can we host @ IEM?

and make PRs to IEM gitlab?

and avoid using githubpages.

> > EDIT: Having the .css file in a fixed URL helps a lot.

> i don't know what that means in this context.

This in the context of editing `.md` files and converting them to `.html` with `pandoc`.
Having a fixed place for the `.css` works in your local machine (when you are previewing/editing) and it also works when the file is in the server.

See https://github.com/Lucarda/Lucarda.github.io/blob/main/pandoc-and-reference/template.txt 

giving the `.css` an URL does the magic.

**Comment by danomatika** [comment]
I agree that info links distributed with the app itself should be limited since they will be “set in stone” to some degree and we do know that old versions of Pd are still in use for many years. It might be convenient to have a Facebook link directly in the app but what happens if FB implodes in 2 years?

I also agree with IOhannes that, while we are using Github right now, we are entirely reliant on the service and can move hosting if/when needed, as we did with Sourceforge.

My suggestion is also to link to a puredata.info page which can keep these more dynamic listings. It’s just a wiki page and can be edited. I am sure IOhannes can give access to those who need it.

enohp ym morf tnes
-----------
Dan Wilcox
danomatika.com
robotcowboy.com

**Comment by danomatika** [comment]
Also, the plone setup at puredata.info is not ideal but I am sure it can be updated, given an updated/modern theme, and reorganized. IMO that makes more sense than dancing around creating an entirely new and parallel set of information while the old pages already have links and search weight.

enohp ym morf tnes
-----------
Dan Wilcox
danomatika.com
robotcowboy.com
